tests/core/models.py

```diff
@@ -24,10 +24,11 @@ class Article(models.Model):
     class Meta:
         app_label = 'core'
 
+
 class User(models.Model):
     name = models.TextField(db_index=True)
     user_id = models.TextField(blank=True, primary_key=True)
-    description = models.TextField(blank=True)
+    about = models.TextField(blank=True)
```

Comment:
Yep, they're really enforcing the field name within same type thing now.

> ### Conflicting field mappings
> Fields with the same name, in the same index, in different types, must have the same mapping, with the exception of the copy_to, dynamic, enabled, ignore_above, include_in_all, and properties parameters, which may have different settings per field.
(https://www.elastic.co/guide/en/elasticsearch/reference/current/breaking_20_mapping_changes.html)